tests/acceptance/owner/not-found-test.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import { test } from 'qunit';
+import moduleForAcceptance from 'travis/tests/helpers/module-for-acceptance';
+import nonExistentOwnerPage from 'travis/tests/pages/owner/non-existent';
+import { percySnapshot } from 'ember-percy';
+
+let adapterException;
+let loggerError;
+
+moduleForAcceptance('Acceptance | owner/not found', {
+  beforeEach() {
+    // Ignore promise rejection.
+    // Original exception will fail test on promise rejection.
+    adapterException = Ember.Test.adapter.exception;
+    loggerError = Ember.Logger.error;
+    Ember.Test.adapter.exception = () => null;
+    Ember.Logger.error = () => null;
+  },
+
+  afterEach() {
+    Ember.Test.adapter.exception = adapterException;
+    Ember.Logger.error = loggerError;
+  }
+});
+
+test('visiting /non-existent-owner shows error message when authenticated', function (assert) {
+  const user = server.create('user');
+  signInUser(user);
+
+  nonExistentOwnerPage.visit();
+
+  andThen(function () {
+    assert.equal(currentURL(), '/non-existent-owner');
+    assert.ok(nonExistentOwnerPage.showsBarricadeIllustration, 'Shows image for aesthetics');
+    assert.equal(nonExistentOwnerPage.errorMessage, 'We couldn\'t find the owner non-existent-owner', 'Shows message that repository was not found');
+    assert.ok(nonExistentOwnerPage.errorMessageProisHidden, 'does not show .com authenticated message');
+  });
+});
+
+test('visiting /non-existent-owner shows error message when unauthenticated', function (assert) {
+  withFeature('proVersion');
+  nonExistentOwnerPage.visit();
+
+  andThen(function () {
+    percySnapshot(assert);
+    assert.equal(currentURL(), '/non-existent-owner');
+    assert.ok(nonExistentOwnerPage.showsBarricadeIllustration, 'Shows image for aesthetics');
+    assert.equal(nonExistentOwnerPage.errorMessage, 'We couldn\'t find the owner non-existent-owner', 'Shows message that repository was not found');
+    assert.ok(nonExistentOwnerPage.errorMessageProUnauthenticated, 'shows .com authenticated message');
```

Comment:
Hopefully this isn’t too nitpicky, but I think the [page object component concept](http://ember-cli-page-object.js.org/docs/v1.8.x/components#default-attributes) could help these be a bit more streamlined. Then these assertions could look like this:

```
assert.ok(nonExistentOwnerPage.barricade.isVisible, …
assert.equal(nonExistentOwnerPage.errorMessage, …
assert.ok(nonExistentOwnerPage.proErrorMessage.isVisible, …
```

That last assertion in the previous test could then be:

```
assert.ok(nonExistentOwnerPage.comErrorMessage.isHidden, …
```

What they call components come with a lot of built-in properties that work in typical situations.